scenarios/post_action_etcd_example.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+pods="$(oc get pods -n openshift-etcd | grep -c Running)"
```

Comment:
I think it would be nice to include a function where the user can place multiple checks within rather than a single command. Multiple checks can be performed and the function return value can be aggregated true/false values from each check.